.github/workflows/test-runner.yml

```diff
@@ -43,6 +55,7 @@ jobs:
           'external'
       }}
     name: Require Approval for External PRs
+    needs: [author-association-member]
```

Comment:
I think you also need to remove the code from line 54?  Or is that for a future PR?